components/03-modules/newsletter/_newsletter.scss

```diff
@@ -0,0 +1,90 @@
+$newsletter__font-size                : $font-size-base !default;
+$newsletter__font-size--agreements    : $font-size-extra-small !default;
+$newsletter__line-height              : 1.2 !default;
+$newsletter__input-background         : #717171 !default;
+$newsletter__input-placholder-opacity : 0.7 !default;
+$newsletter__heading-color            : #ffffff;
```

Comment:
There is defined color `$white` inside `globals/variables_variables.scss`, therefore, you should use this one instead of that.

Reply:
done